### PR TITLE
update ssh key

### PR DESCRIPTION
**Pull request by yuvalbenshalom**
update circleci ssh key

### CODE REVIEW

File: .circleci/config.yml [hunk 1]
```diff
@@ -8,7 +8,7 @@ jobs:
       - setup_remote_docker
       - add_ssh_keys:
           fingerprints:
-            - 2d:1f:7e:83:38:a2:f1:1c:89:86:c3:dd:a7:0e:32:2e
+            - SHA256:HDjXb+EIco/xPtCiblLoN2WDyjfRl8488OsY+JW0EoM
```
Comment (ShacharKidor) [inline .circleci/config.yml:11]: Are we sure it should include the 'SHA256' prefix? it feels strange

**Review by ShacharKidor — CHANGES_REQUESTED** [review]
@yuvalbenshalom - Thanks! 
Please see my question below.

File: .circleci/config.yml (hunk 1 above)
Comment (yuvalbenshalom) [reply .circleci/config.yml:11]: yes.
see example: https://circleci.com/docs/add-ssh-key/

Comment (ShacharKidor) [reply .circleci/config.yml:11]: ok cool. Thanks.

**Review by ShacharKidor — APPROVED** [review]

**Pull request merged**